docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -37,9 +38,9 @@ import WarpStreamSVG from "../../images/logos/warpstream.svg";
 
   ![Fill out connection details](./images/cp_step2.png)
 
-4a. __Optional for Schema Registry__: Specify the complete URL of your Schema Registry server along with the precise RESTful path to the ID representing your preferred schema document. This schema will serve as the validation benchmark for all messages from your topic, consequently blocking any messages that fail the validation against the designated schema. Additionally, the Schema Registry allows new schema retreival via JSON_SR messages. In scenarios where your JSON_SR message carries a Schema Registry ID differing from the current one, Clickpipes will fetch the corresponding schema and use it for future validation.
+4a. __Optional for Schema Registry__: If you are using a Kafka streaming pipe, specify the complete URL of your Schema Registry server along with the precise RESTful path to the ID representing your preferred schema document. This schema will serve as the validation benchmark for all messages from your topic, consequently blocking any messages that fail the validation against the designated schema. Additionally, the Schema Registry allows new schema retreival via JSON_SR messages. In scenarios where your JSON_SR message carries a Schema Registry ID differing from the current one, Clickpipes will fetch the corresponding schema and use it for future validation.
```

Comment:
>  consequently blocking any messages that fail the validation against the designated schema

Is a correct description? Maybe we should provide more ClickPipes-specific details:  
> any messages that fail the validation against the designated schema, will be stored in [the error table](./link_to_error_table.md)`

Reply:
Yeah, honestly I'm not sure after the testing week with the JSON_SR comments I made. There is no rush to merge this PR so I will discuss it in the weekly on Tuesday.

Reply:
> along with the precise RESTful path to the ID representing your preferred schema document.

I believe it's not a strict requirement here. Eventually, the schema ID provided in the URL will be just a fallback. By default, the schema ID provided along with the record is used.

@pjhampton I know it's not a scope of this PR, but maybe we can address it now or when we introduce Avro documentation.

@genzgd thoughts?